docs/results.md

```diff
@@ -108,7 +112,7 @@ high = Queue('high', default_timeout=8)  # 8 secs
 low = Queue('low', default_timeout=600)  # 10 mins
 
 # Individual jobs can still override these defaults
-low.enqueue_call(really_really_slow, timeout=3600)  # 1 hr
```

Comment:
I think here we should use `enqueue()` just like the above snippet as they are in the same section. Besides, `enqueue_call()` usually looks like `enqueue_call(func= really_really_slow)`.

Reply:
Yes, agreed. Could you also replace references to `enqueue_call` with `enqueue`? We really want to encourage people to use `enqueue`

Reply:
@selwin I thought the same. I pushed a new commit replacing `enqueue_call()` with `enqueue()`. PTAL.